src/content/manifest/environment-variables.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Environment variables
+description: Use Okteto's built-in environment variables to streamline your Okteto manifest.
```

Comment:
It's not only `built-in`, we also have user defined variables

Reply:
```suggestion
description: Use variables to streamline your Okteto manifest.
```